src/index.html

```diff
@@ -1,12 +1,363 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#main" class="icon">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
```

Comment:
```suggestion
      <nav class="header__nav nav">
```
header is block. So nav is an element of that block (.header_nav), and its also a block (.nav) . thats why we should have 2 classes here. But we dont use header_nav, so maybe we can ignore it 🤷‍♂️

Reply:
I think I can use the nav block as a block outside of the header